docs/src/domains/quick-start.md

```diff
@@ -62,46 +92,15 @@ highlight=false
 Run the following command:
 
 ```bash
-platform domain:add -p <PROJECT_ID> mysite.com
+platform domain:add -p <PROJECT_ID> <YOUR_DOMAIN>
 ```
 
 {{< /codetabs >}}
 
-As soon as you add the domain, Platform.sh no longer serves `main-def456-abc123.eu-2.platformsh.site` at all.
-Instead, `{default}` in `routes.yaml` is replaced with `mysite.com` anywhere it appears when generating routes to respond to.
-
-You can still access the original internal domain by running `platform environment:info edge_hostname -e <BRANCH_NAME>`.
-
 {{< note >}}
-If you are planning on using subdomains across multiple projects, [the setup will differ slightly](./steps/subdomains.md).
+If you are planning on using subdomains across multiple projects, [the setup differs slightly](./steps/subdomains.md).
 {{< /note >}}
```

Comment:
:question: Is this really relevant on a quickstart? It doesn't seem to be part of the main flow. Perhaps it would fit in the What's next section?